src/duckling/game/path/path-attribute.ts

```diff
@@ -0,0 +1,20 @@
+import {Attribute, Entity} from '../../entitysystem/entity';
+import {Vector} from '../../math/vector';
+
+export const PATH_KEY = "path";
+
+export interface PathAttribute extends Attribute {
+    vertices: Vector[],
+    isLoop: boolean,
+    time: number
```

Comment:
time doesn't tell us if it is one way or two way -- maybe timeToCycle, loopTime, etc.